lib/libClp.jl

```diff
@@ -341,82 +668,186 @@ function Clp_logLevel(model)
     ccall((:Clp_logLevel, libClp), Cint, (Ptr{Clp_Simplex},), model)
 end
 
+"""
+    Clp_lengthNames(model)
+
+length of names (0 means no names0 
+"""
 function Clp_lengthNames(model)
     ccall((:Clp_lengthNames, libClp), Cint, (Ptr{Clp_Simplex},), model)
 end
 
+"""
+    Clp_rowName(model, iRow, name)
+
+Fill in array (at least lengthNames+1 long) with a row name 
+"""
 function Clp_rowName(model, iRow, name)
     ccall((:Clp_rowName, libClp), Cvoid, (Ptr{Clp_Simplex}, Cint, Cstring), model, iRow, name)
 end
 
+"""
+    Clp_columnName(model, iColumn, name)
+
+Fill in array (at least lengthNames+1 long) with a column name 
+"""
 function Clp_columnName(model, iColumn, name)
     ccall((:Clp_columnName, libClp), Cvoid, (Ptr{Clp_Simplex}, Cint, Cstring), model, iColumn, name)
 end
 
+"""
+    Clp_setRowName(model, iRow, name)
+
+Set row name - Nice if they are short - 8 chars or less I think 
+"""
 function Clp_setRowName(model, iRow, name)
     ccall((:Clp_setRowName, libClp), Cvoid, (Ptr{Clp_Simplex}, Cint, Cstring), model, iRow, name)
 end
 
+"""
+    Clp_setColumnName(model, iColumn, name)
+
+Set column name - Nice if they are short - 8 chars or less I think 
+"""
 function Clp_setColumnName(model, iColumn, name)
     ccall((:Clp_setColumnName, libClp), Cvoid, (Ptr{Clp_Simplex}, Cint, Cstring), model, iColumn, name)
 end
 
+"""
+    Clp_initialSolve(model)
+
+General solve algorithm which can do presolve. See ClpSolve.hpp for options
+"""
 function Clp_initialSolve(model)
     ccall((:Clp_initialSolve, libClp), Cint, (Ptr{Clp_Simplex},), model)
 end
 
-function Clp_initialSolveWithOptions(model, arg1)
```

Comment:
This is the only code change. I guess this was an upstream change?

Reply:
Yes, numbering unnamed arguments by position is more reasonable.
See https://github.com/JuliaGeo/GDAL.jl/pull/120#issuecomment-895889617

Reply:
Great!